cinema/tests/test_movie_session_api.py

```diff
@@ -83,7 +83,8 @@ def test_get_movie_sessions_filtered_by_movie(self):
 
     def test_get_movie_sessions_filtered_by_movie_and_data(self):
         movie_sessions = self.client.get(
-            f"/api/cinema/movie_sessions/?movie={self.movie.id}&date=2022-09-2"
+            f"/api/cinema/movie_sessions/?movie="
+            f"{self.movie.id}&date=2022-09-2"
```

Comment:
The date format in the query string should be zero-padded for the day part. Change `2022-09-2` to `2022-09-02` to maintain consistency and avoid potential issues with date parsing.